minecraft/protocol/packet/login.go

```diff
@@ -7,8 +7,9 @@ import (
 // Login is sent when the client initially tries to join the server. It is the first packet sent and contains
 // information specific to the player.
 type Login struct {
-	// ClientProtocol is the protocol version of the player. The player is disconnected if the protocol is
-	// incompatible with the protocol of the server.
+	// ClientProtocol is the protocol version of the player. The player is disconnected if the protocol is incompatible
+	// with the protocol of the server. It is unclear why this is still sent in the login packet, as the ClientProtocol
+	// is already sent in the RequestNetworkSettings packet.
```

Comment:
I wouldn't say it's unclear, I would just say it has been superseded by the protocol version sent in the RequestNetworkSettings packet and that this should no longer be used by the server.